### DISCUSSION TITLE
My Ram Timings for Samsung MGCL and Micron

### DISCUSSION THREAD

**Discussion by hanai3Bi** [hanai3Bi/Switch-OC-Suite#22] [Ram Timing | Customization]
Samsung MGCL : AUTO_ADJ, Vdd2 1150mV, vddq 620mv, 2400Mhz:  5-3-4-4-5-4-5

Micron : AUTO_ADJ, 2400Mhz: 5-4-4-4-5-4-5 (not fully tested)

**Comment by hanai3Bi** [comment]
note that my timings may not work on your console, because diffrent chips have different binnings

**Reply by spriteice** [reply]
Nice work taking over this app. How long would you run memtest and in which mode to consider things stable? Any other stress tests that you could recommend?

**Reply by hanai3Bi** [reply]
> Nice work taking over this app. How long would you run memtest and in which mode to consider things stable? Any other stress tests that you could recommend?

the best test is to run GPU intensive games and see if it freezes or not. some people recommend games like Kirby discovery, Nier, Crash bandicoot,

**Reply by ChanseyIsTheBest** [reply]
Like meha said memtest is surprisingly bad for finding out whether its stable or not because it might be stable for memtest for hours but crash even when loading a game.

Try and find games that max out CPU and GPU at the same time and really push the console to the limit. Nier (in the open world segments) and Kirby and The Forgotten Land with their respective 60FPS cheats will push the console to the limit. Some games also seem more sensitive to memory corruption errors as well but there needs to be wider testing to find more games.

Also probably best to turn of GPU undervolt when you are testing RAM stability because crashes can be caused by GPU being too undervolted.

**Comment by Mgitat** [comment]
These settings work out for my switch oled:
Samsung MGCL : DRAM Timing: 1 Vdd2 1150mV, vddq 625mv, 2400Mhz: 4-2-3-3-4-3-4

I basically started by using hana's Samsung config above but it wasn't stable for my unit, so I reduced each by one and now it works fine.

Huge thanks for this utility, it's amazing and I love tinkering with clocks, timings etc, :D

**Comment by zal16** [comment]
I have a Switch OLED. My timings: Samsung MGCL : Samsung MGCL : DRAM Timing: 1 Vdd2 1137,5mV, vddq 600mv (Default), 2400Mhz: 4-2-3-3-4-3-4

Changing vddq don't helps to incease timing preset numbers for me. But the vdd2 in default voltages, the console won't boot.

I stressed the memory playing Zelda BOTW, GRID Autosport, Crash Bandicoot and Sonic Team Racing, all of them with 60 FPS mods.

It intrigues me that the sonic game runs better than the PS4 version with these overclocks!

**Reply by zal16** [reply]
I had to increase vdd2 voltage to 1150mV, because some sporadic chash still happened.

**Reply by zal16** [reply]
i did more improvements on my timings. I now using 5-5-4-4-5-4-4 (near to the top presets 6-5-5-5-5-4-6)

**Comment by halop** [comment]
OC Suite 1.6.0
Samsung MGCL : AUTO_ADJ, 610mV 1150mV, 2400Mhz: 4-3-4-4-5-5-5
emc_dvb_dvfs_table_t oc_table = { 2400000, { 775, 750, 725, }

**Comment by yoboliu** [comment]
for me (Unknow chips, but prob Micron), the best i can do is:
vdd2:1150000 memclock:2131200
vddq:620000
advanced config:0-4-2-3-3-4-3-4

**Reply by ChanseyIsTheBest** [reply]
You can check your chip type in Hekate. Go to Console Info -> HW & Fuses and it will show your RAM type under DRAM ID.

**Reply by yoboliu** [reply]
thanks, checked and it was samsung.
dam...
